fastlane/Fastfile

```diff
@@ -186,6 +187,16 @@ platform :ios do
     
   end
 
+  desc "Run OWASP Dependency Check"
+  lane :check_dependencies do |options|
+    dependency_check_ios_analyzer(
+      project_name: ENV['REM_FL_TESTS_SLATHER_BASENAME'] || 'Project',
+      output_types: 'html, json',
+      skip_spm_analysis: true,
```

Comment:
SPM analysis didn't work in my local tests.

Reply:
@donnie-jp What was the error?

Reply:
It isn't erroring now but it seems tricky to get both pod and spm analysis working in the same scan. Maybe needs two passes. I'll add this enhancement to tech debt.